dxr/build.py

```diff
@@ -573,7 +576,7 @@ def index_chunk(tree,
                        open_log(tree.log_folder,
                                 'index-chunk-%s.log' % worker_number))
                 for path in paths:
-                    log and log.write('Starting %s.\n' % path)
+                    log and log.write('Starting %s.\n' % path.encode('utf-8'))
```

Comment:
Writing `log and log.write(u'Starting %s.\n' % path)` fails with 'ascii failed to decode ordinal,' see https://travis-ci.org/mozilla/dxr/builds/143472385.